x/auth/signing/adapter.go

```diff
@@ -0,0 +1,60 @@
+package signing
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/protobuf/types/known/anypb"
+
+	txsigning "cosmossdk.io/x/tx/signing"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+)
+
+// V2AdaptableTx is an interface that wraps the GetSigningTxData method.
+// GetSigningTxData returns an x/tx/signing.TxData representation of a transaction for use in signing
+// interoperability with x/tx.
+type V2AdaptableTx interface {
+	GetSigningTxData() txsigning.TxData
+}
+
+// GetSignBytesAdapter returns the sign bytes for a given transaction and sign mode.  It accepts the arguments expected
+// for signing in x/auth/tx and converts them to the arguments expected by the txsigning.HandlerMap, then applies
+// HandlerMap.GetSignBytes to get the sign bytes.
+func GetSignBytesAdapter(
```

Comment:
nit: changelog for public api break

Reply:
This was introduced in https://github.com/cosmos/cosmos-sdk/pull/15822/files#diff-0100bcd16617d3f0e0a8eadcbdf052dfec465cb61c23daea4d201c95c2e20004 so still Unreleased, do we still need a Changelog entry?

Reply:
No, just make sure we update any previous CHANGELOG entry referring to this